sdk/detect/gce/const.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Similar here, these are duplicates of https://github.com/open-telemetry/opentelemetry-go/blob/master/api/standard/resource.go